aliyun/src/test/java/org/apache/iceberg/aliyun/oss/TestOSSOutputFile.java

```diff
@@ -63,11 +63,11 @@ public void testWriteFile() throws IOException {
 
   @Test
   public void testFromLocation() {
-    AssertHelpers.assertThrows(
-        "Should catch null location when creating oss output file",
-        NullPointerException.class,
-        "location cannot be null",
-        () -> OSSOutputFile.fromLocation(ossClient, null, aliyunProperties));
+    Assertions.assertThatThrownBy(
+            () -> OSSOutputFile.fromLocation(ossClient, null, aliyunProperties),
+            "Should catch null location when creating oss output file")
```

Comment:
same here regarding the description part. I don't think it adds any value keeping it